packages/drawer/src/components/Drawer.tsx

```diff
@@ -47,10 +47,6 @@ class Drawer extends React.Component<DrawerProps, DrawerState> {
       overflow: false
     };
 
-    this.bindHandlers();
-  }
-
-  bindHandlers() {
```

Comment:
I'm removing this method because it will allow this method to be used publicly since JS don't have a real private method